app/vmalert/notifier/init.go

```diff
@@ -246,22 +261,54 @@ const (
 // GetTargets returns list of static or discovered targets
 // via notifier configuration.
 func GetTargets() map[TargetType][]Target {
-	var targets = make(map[TargetType][]Target)
-
-	if staticNotifiersFn != nil {
-		for _, ns := range staticNotifiersFn() {
-			targets[TargetStatic] = append(targets[TargetStatic], Target{
-				Notifier: ns,
-			})
-		}
+	if getActiveNotifiers == nil {
```

Comment:
Should it be panic instead? It looks like usage error: GetTargets was called without Init call before.